climada/engine/test/test_impact.py

```diff
@@ -116,7 +116,8 @@ def test_ref_value_insure_pass(self):
         """Test result against reference value"""
         # Read demo entity values
         # Set the entity default file to the demo one
-        ent = Entity.from_excel(ENT_DEMO_TODAY)
+        ent = Entity()
+        ent.read_excel(ENT_DEMO_TODAY)
```

Comment:
This confuses me, it looks like you've changed `read_excel` back to `from_excel`. Is there a reason? Presumably the test passes either way?